src/samples/HandlingCards/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using HandlingCards;
+using Microsoft.Agents.Builder;
+using Microsoft.Agents.Hosting.AspNetCore;
+using Microsoft.Agents.Storage;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System.Threading;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+builder.Services.AddHttpClient();
+
+builder.Logging.AddConsole();
+builder.Logging.AddDebug();
+
+// Register IStorage.  For development, MemoryStorage is suitable.
+// For production Agents, persisted storage should be used so
+// that state survives Agent restarts, and operate correctly
+// in a cluster of Agent instances.
+builder.Services.AddSingleton<IStorage, MemoryStorage>();
+
+// Add AgentApplicationOptions from config.
+builder.AddAgentApplicationOptions();
+
+// Add the Agent
+builder.AddAgent<CardsAgent>();
+
+
+// Configure the HTTP request pipeline.
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+app.UseRouting();
+app.MapPost("/api/messages", async (HttpRequest request, HttpResponse response, IAgentHttpAdapter adapter, IAgent agent, CancellationToken cancellationToken) =>
+    {
+        await adapter.ProcessAsync(request, response, agent, cancellationToken);
+    })
+    .AllowAnonymous();
+
+// Setup port and listening address.
+app.Urls.Add("http://localhost:5000");
```

Comment:
can we use defaults, that can be overriden with launchSettings?

why not 3978?